### PR TITLE
Basque translation for Alsa-Utils

### PR DESCRIPTION
**Pull request by Porrumentzio**
Created `po/eu.po` file and edited `po/LINGUAS` adding basque ISO, `eu`.
_______

I included arrow unicode characters (`←, →, ↑, ↓`) and quotation marks (` ' string ' `) in some translations.
I don't know if this will be a problem. If so, say and I will change them.

### CODE REVIEW

File: po/eu.po [hunk 1]
```diff
@@ -0,0 +1,1725 @@
+# French translation for alsa-utils
```
Comment (fitojb) [inline po/eu.po:1]: Change this to “Basque”

File: po/eu.po [hunk 2]
```diff
@@ -0,0 +1,1725 @@
...
+# Copyright (C) 2011 The ALSA Team
+# This file is distributed under the same license as the alsa-utils package.
+# Christoph J. Thompson <cjsthompson@gmail.com>, 2011.
```
Comment (fitojb) [inline po/eu.po:4]: Replace this line with your name and email, and the current year

File: po/eu.po [hunk 3]
```diff
@@ -0,0 +1,1725 @@
...
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: alsa-utils 1.0.23\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-01-10 09:07+0100\n"
+"PO-Revision-Date: 2020-07-26 12:01+0200\n"
+"Last-Translator: Porrumentzio <porrumentzio@riseup.net>\n"
+"Language-Team: French <fr@li.org>\n"
+"Language: fr\n"
```
Comment (fitojb) [inline po/eu.po:14]: And fix the language code

File: po/eu.po [hunk 4]
```diff
@@ -0,0 +1,1725 @@
...
+# Christoph J. Thompson <cjsthompson@gmail.com>, 2011.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: alsa-utils 1.0.23\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-01-10 09:07+0100\n"
+"PO-Revision-Date: 2020-07-26 12:01+0200\n"
+"Last-Translator: Porrumentzio <porrumentzio@riseup.net>\n"
+"Language-Team: French <fr@li.org>\n"
```
Comment (fitojb) [inline po/eu.po:13]: Change this accordingly

File: po/eu.po [hunk 5]
```diff
@@ -0,0 +1,1725 @@
...
+#, c-format
+msgid "Transfer failed: %s\n"
+msgstr "Bidalketak huts egin du: %s\n"
+
+#: speaker-test/speaker-test.c:1303
+#, c-format
+msgid "Time per period = %lf\n"
+msgstr "Denbora periodoko = %lf\n"
+
+#~ msgid "Warning: format is changed to S16_LE\n"
```
Comment (fitojb) [inline po/eu.po:1718]: Use Poedit to remove these obsolete strings

**Review by fitojb — CHANGES_REQUESTED** [review]

File: po/eu.po (hunk 5 above)
Comment (Porrumentzio) [reply po/eu.po:1718]: Do you suggest me to remove those commented lines? I can do it editing the file, but I don't know how using Poedit.

**Comment by Porrumentzio** [comment]
I see these problems are because there's not any .pot file of which can be started a new translation .po
It would be good if the .pot-s can be added.

**Comment by Porrumentzio** [comment]
It's all done now

**Review by fitojb — APPROVED** [review]

**Comment by perexg** [comment]
Merged in fb2b947d5c828a728aba2e713a4b037bfc675019 . Thanks.

**Pull request closed without merging**